docs/products/cassandra/get-started.md

```diff
@@ -4,34 +4,418 @@ sidebar_label: Get started
+     }
+   }
+
+   provider "aiven" {
+     api_token = var.aiven_token
+   }
+
+   resource "aiven_cassandra" "example_cassandra" {
+     project                 = var.aiven_project_name
+     cloud_name              = "google-europe-west1"
+     plan                    = "startup-4"
+     service_name            = "cassandra-sample"
+     maintenance_window_dow  = "friday"
+     maintenance_window_time = "23:00:00"
+     termination_protection  = false
+
+     cassandra_user_config {
+       migrate_sstableloader = true
+       service_log           = false
+     }
+   }
+
+   output "cassandra_service_host" {
+     value = aiven_cassandra.example_cassandra.service_host
+   }
+
+   output "cassandra_service_port" {
+     value = aiven_cassandra.example_cassandra.service_port
+   }
+
+   output "cassandra_service_username" {
+     value = aiven_cassandra.example_cassandra.service_username
+   }
+
+   output "cassandra_service_password" {
+     value     = aiven_cassandra.example_cassandra.service_password
+     sensitive = true
+   }
+    ```
+
+1. Create the `terraform.tfvars` file for assigning actual values to your previously
+   declared variables.
+
+   ```hcl
+   aiven_token    = "AIVEN_TOKEN"
+   aiven_project_name = "PROJECT_NAME"
+   ```
+
+1. Run `terraform init` > `terraform plan` > `terraform apply`.
+
+1. Store Terraform outputs in environment variables so that they can be used for
+   [connecting](#connect-to-service):
+
+   ```bash
+   CASSANDRA_HOST="$(terraform output -raw cassandra_service_host)"
+   CASSANDRA_PORT="$(terraform output -raw cassandra_service_port)"
+   CASSANDRA_USER="$(terraform output -raw cassandra_service_username)"
+   CASSANDRA_PASSWORD="$(terraform output -raw cassandra_service_password)"
+   ```
+
+</TabItem>
+<TabItem value="3" label="K8s">
+
+Create an Aiven for Apache Cassandra service using the Aiven Operator for Kubernetes.
+
+1. [Get authenticated and authorized](https://aiven.github.io/aiven-operator/authentication.html).
+1. Create file `cassandra-sample.yaml` with the following content:
+
+    ```yaml
+    apiVersion: aiven.io/v1alpha1
+    kind: Cassandra
+    metadata:
+      name: cassandra-sample
+    spec:
+      authSecretRef:
+        name: aiven-token
+        key: token
+
+      connInfoSecretTarget:
+        name: cassandra-secret
+
+      userConfig:
+        migrate_sstableloader: true
+        service_log: false
+
+      project: PROJECT_NAME
+      cloudName: google-europe-west1
+      plan: startup-4
+
+      maintenanceWindowDow: friday
+      maintenanceWindowTime: 23:00:00
+    ```
+
+1. Create the service by applying the configuration:
+
+   ```shell
+   kubectl apply -f cassandra-sample.yaml
+   ```
+
+1. Review the resource you created with the following command:
+
+   ```shell
+   kubectl get cassandras cassandra-sample
+   ```
+
+The resource can stay in the `REBUILDING` state for a couple of minutes. Once the state
+changes to `RUNNING`, you are ready to access it.
```

Comment:
We do not say this for TF, but I suppose it's equally correct and should appear in the TF tab too?